File: 5CalculateAvg.js
//my solution
function findAverage(array) {
    if (array.length == 0){
      return 0;
    } else {
      summ = 0
      for (let i = 0; i < array.length; i++){
        summ += array[i]
      }
      return summ / array.length
    }
  }


// most upvoted solution
function find_average(array) {
    if (array.length === 0) {
    return 0;
    }
    var result = 0;
    for (i=0; i<array.length; i++) {
      result +=array[i];
    }
    return result/array.length;
  }

File: 5CalculateAvg.py
def find_average(numbers):
    summ = 0
    if (len(numbers) == 0):
        return 0
    else:
        for num in numbers:
            summ += num  
    return summ / len(numbers)
    pass

# I confused myself to divide averages with adding the two and dividing by 2
# it's interesting to know that python can do sum of an array per solution below

def find_average(array):
    try:
        return sum(array) / len(array)
    except ZeroDivisionError:
        return 0